src/components/People.tsx

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable prefer-template */
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import { useState } from 'react';
+import { getPeople } from '../api';
+import { Loader } from './Loader';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+export const People: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>();
+  const [isClicked, setIsClicked] = useState<string | null>('');
+  const [isError, setIsError] = useState<string | null>(null);
+
+  getPeople().then((res) => {
+    setPeople(res);
+  }).catch((error) => {
+    setIsError(error);
+  });
```

Comment:
this should be done in an `useEffect` ❗ ❗ ❗ 

or in some kind of event, but NEVER just fetch items inside your comonent directly

Reply:
+1